wbooks/src/constants/fonts.ts

```diff
@@ -0,0 +1,6 @@
+export const FONTS = {
```

Comment:
FONTS const it's not necessary (the file is called fonts). You can declare a object SIZE and export that.

Reply:
Done!